app/__init__.py

```diff
@@ -20,7 +20,7 @@ def create_app():
     )
 
     from app.config import configs
-    environment = os.getenv('NOTIFY_ENVIRONMENT', 'development')
+    environment = os.getenv('HOST', 'development')
```

Comment:
Shouldn't this default to "local", since development is now not an option for the host?